android/app/build.gradle

```diff
@@ -92,13 +92,13 @@ def enableSeparateBuildPerCPUArchitecture = false
 def enableProguardInReleaseBuilds = false
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion 26
+    buildToolsVersion '26.0.1'
```

Comment:
I'll be pedantic, but let's change the quotes to double, as it used to be? :smile:

Reply:
** curses Android Studio **

Reply:
Heh, thank you :+1: